features/healthcheck

```diff
@@ -0,0 +1,36 @@
+.. _feature_healthcheck:
+
+$healthstatus, Health Check
+===========================
+
+Description
+-----------
+
+For a quick check to see if Firely Server is up and running, it provides the ``$healthstatus`` custom operation. You invoke it at the system level, so at the root:
```

Comment:
FHIR operations could be either GET or POST. We should add a hint that this is in fact a GET request.